shared/embeds/index.js

```diff
@@ -66,6 +67,13 @@ export default [
     component: Airtable,
     matcher: matcher(Airtable),
   },
+  {
+    title: "Bilibili",
+    keywords: "video",
```

Comment:
Include the Chinese in keywords?

Reply:
Is it like this? Does it feel strange? I'm not sure what to do.

>     keywords: "video, 哔哩哔哩, 视频弹幕网站",


Are there any references and standards for this type of integration? Maybe there should be a catalog or website about outline development and design specifications.